sprocs/assessment_instances_select_log.sql

```diff
@@ -124,6 +124,8 @@ BEGIN
                     s.id AS log_id,
                     jsonb_build_object(
                       'submitted_answer', CASE WHEN include_files THEN s.submitted_answer ELSE (s.submitted_answer - '_files') END,
+                      'raw_submitted_answer', CASE WHEN include_files THEN s.raw_submitted_answer
+                      ELSE (SELECT JSONB_OBJECT_AGG(key, value) FROM JSONB_EACH(s.raw_submitted_answer) WHERE key !~ '^_') END,
```

Comment:
Nit: I think it would be more efficient to use `starts_with()` (or the completely unintuitive `^@` operator) instead of a regex here.

Reply:
Done. Do you think I should do the same treatment for `submitted_answer`, since it will (for elements like `pl-file-editor`) retain that same raw input?

Reply:
Sorry, I'm not quite following here. What extra raw input is retained in `submitted_answer`?

Reply:
<img width="575" alt="image" src="https://user-images.githubusercontent.com/1004907/197239535-268ae8d2-f50b-43ea-a53c-6bc22214080c.png">

Note that this is `submitted_answer`. `pl-file-editor` (and `pl-rich-text-editor` for that matter) transfer this content to the `_files` object, but also keep it in the original raw form. `pl-file-upload` removes this entry.

https://github.com/PrairieLearn/PrairieLearn/blob/1ab3a6e7accfc4ff4245198b794a3cf19cff6e3b/elements/pl-file-upload/pl-file-upload.py#L82-L85

Though another approach would be, of course, to remove this entry in these other elements as well...

Reply:
Sounds good. The elements rely on this for rendering now, I'll add an issue about it.